filecoin-proofs/src/constants.rs

```diff
@@ -4,17 +4,21 @@ use std::sync::RwLock;
 use lazy_static::lazy_static;
 use storage_proofs::hasher::Hasher;
 use storage_proofs::util::NODE_SIZE;
+use typenum::{U0, U2, U8};
 
 use crate::param::{ParameterData, ParameterMap};
 use crate::types::UnpaddedBytesAmount;
 
 pub const SECTOR_SIZE_2_KIB: u64 = 2_048;
+pub const SECTOR_SIZE_4_KIB: u64 = 2 * SECTOR_SIZE_2_KIB;
+pub const SECTOR_SIZE_16_KIB: u64 = 8 * SECTOR_SIZE_2_KIB;
+pub const SECTOR_SIZE_32_KIB: u64 = 8 * SECTOR_SIZE_4_KIB;
 pub const SECTOR_SIZE_8_MIB: u64 = 1 << 23;
+pub const SECTOR_SIZE_16_MIB: u64 = 2 * SECTOR_SIZE_8_MIB;
 pub const SECTOR_SIZE_512_MIB: u64 = 1 << 29;
+pub const SECTOR_SIZE_1_GIB: u64 = 2 * SECTOR_SIZE_512_MIB;
 pub const SECTOR_SIZE_32_GIB: u64 = 1 << 35;
-
-pub const ELECTION_POST_CHALLENGE_COUNT: usize = 65;
-pub const ELECTION_POST_CHALLENGED_NODES: usize = 1;
+pub const SECTOR_SIZE_64_GIB: u64 = 2 * SECTOR_SIZE_32_GIB;
 
```

Comment:
Eventually, let's clean up our theory of defining these constants to be less arbitrary.